src/site/topic16.rst

```diff
@@ -8,6 +8,36 @@ Runtime Errors
 Throwing an Exception
 =====================
 
+* We have seen multiple examples of throwing an exception
```

Comment:
Exception causes are also pretty useful, though moreso on the maintenance side of things. Probably not worth going into?

Reply:
Perhaps another aside. Defo out of scope, but still something they could be made aware of at this stage.